js-api-spec/value/calculation.test.ts

```diff
@@ -216,9 +212,146 @@ describe('SassCalculation', () => {
       }
     });
   });
+
+  describe('simplifies', () => {
+    it('calc()', () => {
+      const fn = () =>
+        SassCalculation.calc(
+          new CalculationOperation('+', new SassNumber(1), new SassNumber(2))
+        );
+
+      expect(
+        compileString('a {b: foo()}', {
+          functions: {'foo()': fn},
+        }).css
+      ).toBe('a {\n  b: 3;\n}');
+    });
+
+    it('clamp()', () => {
+      const fn = () =>
+        SassCalculation.clamp(
+          new SassNumber(1),
+          new SassNumber(2),
+          new SassNumber(3)
+        );
+
+      expect(
+        compileString('a {b: foo()}', {
+          functions: {'foo()': fn},
+        }).css
+      ).toBe('a {\n  b: 2;\n}');
+    });
+
+    it('min()', () => {
+      const fn = () =>
+        SassCalculation.min([new SassNumber(1), new SassNumber(2)]);
+
+      expect(
+        compileString('a {b: foo()}', {
+          functions: {'foo()': fn},
+        }).css
+      ).toBe('a {\n  b: 1;\n}');
+    });
+
+    it('max()', () => {
+      const fn = () =>
+        SassCalculation.max([new SassNumber(1), new SassNumber(2)]);
+
+      expect(
+        compileString('a {b: foo()}', {
+          functions: {'foo()': fn},
+        }).css
+      ).toBe('a {\n  b: 2;\n}');
+    });
+
+    it('operations', () => {
+      const fn = () =>
+        SassCalculation.calc(
+          new CalculationOperation(
+            '+',
+            SassCalculation.min([new SassNumber(3), new SassNumber(4)]),
+            new CalculationOperation(
+              '*',
+              SassCalculation.max([new SassNumber(5), new SassNumber(6)]),
+              new CalculationOperation(
+                '-',
+                new SassNumber(3),
+                new CalculationOperation(
+                  '/',
+                  new SassNumber(4),
+                  new SassNumber(5)
+                )
+              )
+            )
+          )
+        );
+
+      expect(
+        compileString('a {b: foo()}', {
+          functions: {'foo()': fn},
+        }).css
+      ).toBe('a {\n  b: 16.2;\n}');
+    });
+  });
+
+  const primitiveValues: [any, string][] = [
+    [new SassString('quoted', {quotes: true}), '"quoted"'],
+    [new SassString('unquoted', {quotes: false}), 'unquoted'],
+    [new SassNumber(1), '1'],
+    [sassTrue, 'true'],
+    [sassFalse, 'false'],
+  ];
+  describe('does not simplify', () => {
+    for (const [value, output] of primitiveValues) {
+      it(value.toString(), () => {
+        expect(
+          compileString('a {b: foo()}', {
+            functions: {'foo()': () => value},
+          }).css
+        ).toBe(`a {\n  b: ${output};\n}`);
+      });
+    }
+  });
```

Comment:
I'm not sure how much value this test is adding. If it's trying to assert that all non-`Calculation`s are passed through unaltered, this doesn't cover all the Sass value types and would be annoying to keep up-to-date even if it did. Otherwise, it seems like the general principle that "a normal Sass value is returned as-is" should be exercised fairly thoroughly be the existing function tests.